src/components/StepZero.jsx

```diff
@@ -34,40 +34,35 @@ class StepZero extends Component {
 					<p className="mt-3">This is also available in <a href="#" role="link">Tagalog</a>, more regional
 						languages will be available soon.</p>
 				</div>
-				<p>
-					If you have no symptoms or exposure concerns, but have questions about COVID-19 or novel
-					coronavirus, please visit the following resources:
-				</p>
 				<div>
 					<p>If you have no symptoms or exposure concerns, but have questions about COVID-19 or novel
 						coronavirus,
 						please visit the following resources:</p>
 					<ul>
-						<li><a href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019">World Health
+						<li><a href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019" role="link">World Health
 							Organization</a> – World Health
 							Organization’s official webpage regarding COVID-19
 						</li>
-						<li><a href="https://www.doh.gov.ph/2019-nCoV">Department of Health</a> – Learn about
+						<li><a href="https://www.doh.gov.ph/2019-nCoV" role="link">Department of Health</a> – Learn about
 							COVID-19
 							and actions being taken to protect the health of Filipinos
 						</li>
 						<li>Department of Health’s official social media channels:
 							<ul>
-								<li><a href="https://www.facebook.com/OfficialDOHgov">Facebook</a></li>
-								<li><a href="https://twitter.com/DOHgovph">Twitter</a></li>
-								<li><a href="http://bit.ly/DOHPhCOVID-19">Viber</a></li>
+								<li><a href="https://www.facebook.com/OfficialDOHgov" role="link">Facebook</a></li>
```

Comment:
what's `role=link` for?

Reply:
ARIA tags. since "Facebook" lang ang content ng <a>

Reply:
tag

Reply:
TIL